frontend/src/components/maps/ParcelPopupView.tsx

```diff
@@ -57,14 +59,13 @@ export const ParcelPopupView = (props: IParcelDetailProps | null) => {
             </ListGroup>
           </Col>
           <Col>
-            {parcelDetail?.id && !props?.disabled && (
-              <Link to={`/submitProperty/${parcelDetail?.id}?disabled=true`}>View</Link>
-            )}
-            {parcelDetail?.id && !props?.disabled && (
-              <Link style={{ marginLeft: '8px' }} to={`/submitProperty/${parcelDetail?.id}`}>
-                Update
-              </Link>
-            )}
+            {parcelDetail?.id &&
+              !props?.disabled &&
+              (!keycloak.hasAgency(parcelDetail?.agencyId) ? (
+                <Link to={`/submitProperty/${parcelDetail?.id}?disabled=true`}>View</Link>
+              ) : (
+                <Link to={`/submitProperty/${parcelDetail?.id}`}>Update</Link>
+              ))}
```

Comment:
The logic here seems a bit hard to follow to me. 
What would you think of maybe having two booleans `canView`, `canUpdate` and then JSX looks a bit cleaner IMO?

```jsx
{canView && (<Link ... />)}
{canUpdate && (<Link ... />)}
```